eidpauth/src/main/java/uk/ac/ucl/eidp/auth/AuthController.java

```diff
@@ -0,0 +1,81 @@
+package uk.ac.ucl.eidp.auth;
+
+import java.security.Principal;
+import java.util.Optional;
+import uk.ac.ucl.eidp.auth.jaas.SingleFactorCallbackHandler;
+import uk.ac.ucl.eidp.auth.jwt.Jwt;
+import uk.ac.ucl.eidp.auth.model.RoleE;
+import uk.ac.ucl.eidp.auth.model.UserE;
+
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import javax.ejb.EJB;
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+import javax.security.auth.Subject;
+import javax.security.auth.callback.CallbackHandler;
+import javax.security.auth.login.LoginContext;
+import javax.security.auth.login.LoginException;
+
+/**
+ *
+ * @author David Guzman {@literal d.guzman at ucl.ac.uk}
+ */
+@Stateless
+public class AuthController implements AuthControllerLocal {
+
+  @EJB
+  UserController userController;
+  
+  @Inject
+  Jwt jwt;
+  
+  LoginContext loginContext;
+
+  /**
```

Comment:
\[Checkstyle\] WARNING: First sentence of Javadoc is incomplete \(period is missing\) or not present\.